layers/xt-prod-devel-rpi5-domd/recipes-core/image/rpi5-image-xt-domd.bbappend

```diff
@@ -10,10 +10,16 @@ WIFI_PACKAGES = "			\
     kernel-module-brcmfmac		\
 "
 
+CAN_PACKAGES = "			\
+    iproute2				\
```

Comment:
Are you sure that iproute is needed when can is enabled? And if yes please mention it in the commit description,